react/src/components/Chat/AssistantChatMesssage.tsx

```diff
@@ -0,0 +1,34 @@
+import ChatMessage, { ChatMessageProps } from './ChatMessage';
+import CopyButton from './CopyButton';
+import Compact from 'antd/es/space/Compact';
+
+export const AssistantChatMessage: React.FC<ChatMessageProps> = ({
```

Comment:
Add new component for each role, user, assistant derived from ChatMessage, which make more clear component role and clean code for props